login/resources/css/login.css

```diff
@@ -604,15 +606,10 @@ form#kc-select-back-form div.login-pf-social-section {
   width: 150px;
   height: 60px;
   margin: 40px auto 0;
-  padding: 0
+  padding: 0;
+  margin-top: 50px;
```

Comment:
Can this be dynamic and based on a viewport height? So if the page is zoomed in - everything is still visible without scrolling down